10:38:04 > An error occurred while executing the command definition. See the inner exception for details.
16:58:28 > Error al reflejar el tipo 'EFModelo.PECAPE'.
17:10:55 > Error al reflejar el tipo 'EFModelo.PECAPE'.
17:20:37 > Se produjo una excepción en el destino de la invocación.
17:20:40 > Se produjo una excepción en el destino de la invocación.
17:20:50 > No se puede reconocer la cadena como valor DateTime válido.
